crates/swc_ecma_transforms_module/tests/fixture/common/issue-4988/3/output.amd.js

```diff
@@ -0,0 +1,9 @@
+define([
+    "require",
+    "foo"
+], function(require, _foo) {
+    "use strict";
+    _foo.bar = 1;
+    module.exports = _foo;
```

Comment:
Expect `export = expr` to transform to `return expr`.
For similar reasons to preserve `import =`, we should preserve `export =`.

Reply:
In AMD? I think it makes sense.

Reply:
> In AMD? I think it makes sense.

`import=` implies nointerop.

---
`export = ` will overwrite all the exports.

In AMD, we should return it in the end. [TS REPL](https://www.typescriptlang.org/play?jsx=1&module=2&sourceMap=true#code/JYWwDg9gTgLgBAMwhRUIjgIiRTBuAKFEljgCMBDKOAXjigFMBHAV2EYApNKpMBKQgRwcBBHiMEAbBvEoAvWnACMhBgA8S8OvMHzFAJjxA).

It's similar in [CJS](https://www.typescriptlang.org/play?jsx=1&module=1&sourceMap=true#code/JYWwDg9gTgLgBAMwhRUIjgIiRTBuAKFEljgCMBDKOAXjigFMBHAV2EYApNKpMBKQgRwcBBHiMEAbBvEoAvWnACMhBgA8S8OvMHzFAJjxA).

---

I think we can allow this option in both AMD and CJS module transforms.

Reply:
Yeah I think you are right